app/src/main/scala/org/alephium/explorer/util/Scheduler.scala

```diff
@@ -117,7 +117,9 @@ class Scheduler private (name: String, timer: Timer, @volatile private var termi
       case Failure(exception) =>
         //Log the failure.
         logger.error(s"${logId(taskId)}: Failed executing task", exception)
-        scheduleLoop(taskId, loopInterval, loopInterval)(block)
+        if (!terminated) {
+          scheduleLoop(taskId, loopInterval, loopInterval)(block)
+        }
```

Comment:
git extremist here: This change doesn't seems to be related to that commit message :)

Reply:
Ah my bad. I was being lazy. Will make sure next time :)